python_modules/dagster/dagster/core/types/__init__.py

```diff
@@ -1,20 +1,38 @@
-from .base import DagsterType, DagsterTypeAttributes
+from .builtin_enum import BuiltinEnum
+from .dagster_type import Nullable, List
+from .field import Dict, Field, NamedDict
+from .runtime import PythonObjectType
 
-from .builtins import (
-    Any,
-    Bool,
-    DagsterStringType,
-    Dict,
-    Int,
-    List,
-    NamedDict,
-    Nullable,
-    Path,
-    PythonDict,
-    PythonObjectType,
-    String,
-)
+Any = BuiltinEnum.ANY
+String = BuiltinEnum.STRING
+Int = BuiltinEnum.INT
+Bool = BuiltinEnum.BOOL
+Path = BuiltinEnum.PATH
```

Comment:
i'm uneasy about the builtin path, as somehow not rich enough... for whatever reason this feels like a user-defined or contrib type